.php-cs-fixer.dist.php

```diff
@@ -11,15 +11,16 @@
 ;
 
 // Indentation inside switch blocks and some multiline
-// elements (control statements, assignments, strings) 
+// elements (control statements, assignments, strings)
 // is not considered. See:
 // https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/776
 // https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4502
 //
 // PhpCsFixer's method_argument_space default value,
 // which includes ensure_fully_multiline, causes
 // inconsistencies in templates.
-return PhpCsFixer\Config::create()
+$config = new PhpCsFixer\Config();
+return $config
```

Comment:
Notpick: you could chain the constructor using parenthesis ...

```php
return (new PhpCsFixer\Config())
    ->setRules([...
```

https://www.php.net/manual/en/language.oop5.basic.php#example-173

Reply:
@jraddaoui okay, I've implemented your suggestion in more recent commits.